### DISCUSSION TITLE
Which metrics can be captured by Kindling?

### DISCUSSION THREAD

**Discussion by discaca** [harmonycloud/kindling#136] [agent probe] [answered]
Which metrics can be captured by Kindling? And how to use them?

**Comment by dxsup** [comment, marked as answer]
Documentation about what metrics the agent emits is at [here](http://www.kindling.space:33215/project-1/doc-15/). You could refer to the section [PQL Example](http://www.kindling.space:33215/project-1/doc-15/#h1-pqlexample) to find the basic use cases. We recommend users install the Grafana dashboards to make the most use of our metrics, from which you could also learn how to use them.
